pypit/arflux.py

```diff
@@ -1,4 +1,7 @@
-# Module for fluxing routines
+""" Module for fluxing routines
+"""
+from __future__ import (print_function, absolute_import, division,
```

Comment:
Definitely a fair point.  I'm starting to sprinkle these in to
push us toward Python 3 compliance.   I believe the community
is headed that way, and so shall we (need to be)...